charts/victoria-logs-single/templates/server-ingress.yaml

```diff
@@ -20,6 +20,11 @@ spec:
   {{- end }}
   rules:
     {{- range $app.ingress.hosts }}
+    {{- $port := dict "name" "http" }}
+    {{- with .port }}
+    {{- $numberTypes := list "int" "float64" }}
+    {{- $port = dict (ternary "number" "name" (has (kindOf .) $numberTypes)) . }}
+    {{- end }}
```

Comment:
Seems like this part is quite repetitive, and we now have `common` chart. Would that make sense to move it there, wdyt?

Reply:
moved logic to a common chart, added values for chart's ingress linting